### PR TITLE
Improve readability of main module

### PR DESCRIPTION
**Pull request by ashmaroli**
(explanations provided with commit messages)

### CODE REVIEW

File: lib/jekyll/webmention_io.rb [hunk 1]
```diff
@@ -351,13 +351,14 @@ def self.uri_ok?(uri)
...
+def require_all(group)
+  Dir[File.expand_path("#{group}/*.rb", __dir__)].each do |file|
+    require file
+  end
 end
+
+require_all "commands"
+require_all "generators"
+
+require_relative "tags/_.rb"
```
Comment (aarongustafson) [inline lib/jekyll/webmention_io.rb]: Will Ruby automatically skip requiring `_.rb` a second time when `require_all` is called and the directory is looped?

Comment (ashmaroli) [reply lib/jekyll/webmention_io.rb]: Yes, the `require*` macros are optimized that way by checking against a global object `$LOADED_FEATURES`

Comment (aarongustafson) [reply lib/jekyll/webmention_io.rb]: Ok cool.

**Comment by aarongustafson** [comment]
Thanks for all of this cleanup!

**Pull request merged**